apps/studio/components/interfaces/TableGridEditor/DeleteConfirmationDialogs.tsx

```diff
@@ -152,14 +148,15 @@ const DeleteConfirmationDialogs = ({
 
   const onConfirmDeleteColumn = async () => {
     if (!(snap.confirmationDialog?.type === 'column')) return
+    if (project === undefined) return
 
     const selectedColumnToDelete = snap.confirmationDialog.column
     if (selectedColumnToDelete === undefined) return
 
     deleteColumn({
       id: selectedColumnToDelete.id,
       cascade: isDeleteWithCascade,
-      projectRef: projectRef!,
+      projectRef: project?.ref,
```

Comment:
This so doesn't matter but I can't unsee... 🙈

because there's `if (project === undefined) return` above, this can be this right?:

```suggestion
      projectRef: project.ref,
```